docs/Grade.md

```diff
@@ -0,0 +1,58 @@
+# Grade Component
+
+The `Grade` component is a customizable avatar-based component that represents different grades. It allows you to display a label with a specified background color and font color.
+
+## Usage
+
+```tsx
+import { Grade } from '@brightlayer-ui/react-native-components';
+
+<Grade label={'CS'} fontColor="#FFFFFF" backgroundColor="#198900" size={40} />;
+
+```
+
+### Grade API
+
+| Prop Name           | Description                                | Type                                   | Required | Default                |
+| ------------------- | ------------------------------------------ | -------------------------------------- | -------- | ---------------------- |
+| label               | The Text shown in Grade                    | String                                 | yes      |                        |
+| fontColor           | The color of the text label.               | string                                 | yes      |                        |
+| backgroundColor     | The Background color of Grade              | string                                 | yes      |                        |
+| size                | The radius of Grade component              | number                                 | yes      |                        |
+
+
+## SubComponent
+
+The Subcomponents gives you a set of preconfigured components that can directly be used 
+
+### USAGE
+
+```tsx
+import Grade from '@brightlayer-ui/react-native-components';
+<View> 
+	<Grade.APlus /> {/* Renders A+ grade */} 
+	<Grade.A /> {/* Renders A grade */} 
+	<Grade.AMinus /> {/* Renders A- grade */}
+	<Grade.BPlus/> {/* Renders B+ grade */}
+	<Grade.B/> {/* Renders B grade */}
+	<Grade.BMinus/> {/* Renders B- grade */}
+	<Grade.CPlus/> {/* Renders C+ grade */}
+	<Grade.C/> {/* Renders C grade */}
+	<Grade.CMinus/> {/* Renders C- grade */}
+	<Grade.DPlus/> {/* Renders D+ grade */}
+	<Grade.D/> {/* Renders D grade */}
+	<Grade.DMinus/> {/* Renders D- grade */}
+	<Grade.F/> {/* Renders F grade */}
+</View>
+```
+### Grade API
+
+All Grade Subcomponents have the following Props
+
+| Prop Name           | Description                                | Type                                   | Required | Default                |
+| ------------------- | ------------------------------------------ | -------------------------------------- | -------- | ---------------------- |
+| size                | The radius of Grade component              | number                                 | No       | 40                     |
+
+### Styles
+
+You can override the internal style used by Brightlayer UI by passing a `style` prop. It supports all the View Props on Text Container
```

Comment:
"all the View Props on Text Container" is there a list of all these props that we can link to?

Reply:
Also, what did you mean by "Text Container" here?